packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/SignUpAndSignInDiffSection/DiffSegment.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  children: ReactNode;
+  hasChanged: boolean;
+  isAfter?: boolean;
+};
+
+const DiffSegment = ({ children, hasChanged, isAfter = false }: Props) => {
+  if (!hasChanged) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <>{children}</>;
```

Comment:
will return children do? not sure

Reply:
If we return children, we'll get an error:
<img width="528" alt="image" src="https://user-images.githubusercontent.com/10806653/199388687-a9015f29-f35c-43d4-b659-632c7acce2bf.png">